tests/output/test_bazefetcher.py

```diff
@@ -28,6 +28,13 @@ def assert_correct_index(expected_index, basedir, start_date, end_date,
     assert (result.index == expected_index).all()
 
 
+def assert_correct_raw_values(expected_series, basedir, file_date, tag="test"):
```

Comment:
Not entirely sure about this function name. Perhaps it should just be called assert_correct_ordering?

Reply:
Well, in theory it doesn't assert only ordering. It asserts raw values that are written to file and not processed by bazefetcher source. If there was some other issue, like additional border value written to file, that bazefetcher source strips off, the code would catch it, but it won't be an ordering issue.

Reply:
Ok, I'm convinced